src/applications/gqrx/receiver.cpp

```diff
@@ -888,11 +890,13 @@ receiver::status receiver::set_demod(rx_demod demod, bool force)
     case RX_DEMOD_WFM_S:
         connect_all(RX_CHAIN_WFMRX);
         rx->set_demod(wfmrx::WFMRX_DEMOD_STEREO);
+        d_is_stereo_type_demodulator = true;
         break;
 
     case RX_DEMOD_WFM_S_OIRT:
         connect_all(RX_CHAIN_WFMRX);
         rx->set_demod(wfmrx::WFMRX_DEMOD_STEREO_UKW);
+        d_is_stereo_type_demodulator = true;
         break;
 
     case RX_DEMOD_SSB:
```

Comment:
`RX_DEMOD_NONE` produces 2 channels and needs `d_is_stereo_type_demodulator = true`  (it is raw IQ)